### PR TITLE
feat: add some UI updates

### PR DESCRIPTION
**Pull request by Mubangizi**
# Description
Affected pages
- Auth pages
- Project Settings Page
- App setting page

### CODE REVIEW

File: src/components/PrimaryButton/index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import PropTypes from "prop-types";
 const PrimaryButton = (props) => {
   const { children, className, btntype, color, small, transparent, noPadding } =
     props;
-
+  console.log(color);
```
Comment (rhodinemma) [inline src/components/PrimaryButton/index.js]: this sneaked in 😁
should be removed.

**Review by rhodinemma — COMMENTED** [review]
thank you for these impactful changes @Mubangizi 
upon review i noticed that the app revisions section was imbalanced for more than one row and Select component under image settings in the App settings could be aligned too!

Screenshot 2024-11-27 at 11 17 03 AM

File: src/components/PrimaryButton/index.js (hunk 1 above)
Comment (Mubangizi) [reply src/components/PrimaryButton/index.js]: This has been resolved

**Comment by Mubangizi** [comment]
> thank you for these impactful changes @Mubangizi upon review i noticed that the app revisions section was imbalanced for more than one row and Select component under image settings in the App settings could be aligned too!
> 
> Screenshot 2024-11-27 at 11 17 03 AM

I have updated the app revisions bit. As for the select Input, and all other input secetions I havent touched those maybe I will work with them later. If I have the time

**Review by rhodinemma — APPROVED** [review]
thank you for this @Mubangizi 🎉

**Pull request merged**